draft-ietf-ice-pac.xml

```diff
@@ -172,37 +179,41 @@
 
     <section title="Update to RFC 8445">
       <t>
-        In order to avoid the problem raised by this document, the ICE agent
-        needs to wait enough time to allow peer-reflexive candidates to be
-        discovered. Accordingly, when a full ICE implementation begins its
-        ICE processing, as described in <xref target="RFC8445" />, Section
-        6.1, it MUST set a timer, and its duration SHOULD be
-        equal to the agent's connectivity check transaction timeout.
+        In order to avoid the problem described in this specification, this
+        section updates <xref target="RFC8445" />. It defines a new timer, Tpac,
+        and defines new procedures associated with the timer for full ICE implementations.
+      </t>
```

Comment:
It seems like this paragraph says the exact same thing as the previous paragraph, and we could probably just remove this one.

Reply:
Since this is the Update-to-8445 section, I want to give a description of what is updated - especially since we don't do the old-text-new-text type of update.